include/umf/mempolicy.h

```diff
@@ -25,17 +25,30 @@ typedef enum umf_mempolicy_membind_t {
     /// Bind memory to namespace
     UMF_MEMPOLICY_BIND,
     /// Prefer memory from namespace but fallback to other memory if not available
-    UMF_MEMPOLICY_PREFERRED
+    UMF_MEMPOLICY_PREFERRED,
+    /// Allocation will be split evenly across nodes specified in nodemask.
+    /// umf_numa_split_partition_t can be passed in umf_os_memory_provider_params_t structure
```

Comment:
did you mean `umf_mempolicy_split_partition_t`? and used in `umfMempolicySetCustomSplitPartitions`?